monitoring/install-ks-monitoring.sh

```diff
@@ -175,4 +234,4 @@ wait-for-object $ctx $ns statefulset pyroscope
 wait-for-object $ctx $ns statefulset pyroscope-alloy
 
 # Delete temporary file
-rm /tmp/pyroscope-config-values-set.yaml
\ No newline at end of file
+rm /tmp/pyroscope-config-values-set.yaml
```

Comment:
There is more to remove now, right?

Reply:
Could you provide more details? I don't understand this comment. At each stage of deployment of the components "temporary files" are being deleted.